website/pages/blog/optimize-bundle-size-with-swc-plugin-for-graphql-codegen.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Optimize Bundle Size with SWC Plugin for GraphQL Code Generator
+authors: jessevelden
+tags: [client-preset, codegen]
+date: 2023-02-24
+description:
+  'Optimize your Bundle Size with the SWC plugin for client-preset for GraphQL Code Generator'
+# image: /blog-assets/optimize-bundle-size-with-swc-plugin-for-graphql-codegen/coverTODO.png
+# thumbnail: /blog-assets/optimize-bundle-size-with-swc-plugin-for-graphql-codegen/thumbnailTODO.png
+---
+
+With the new
+[`client-preset` for GraphQL Code Generator](https://graphql-code-generator.com/plugins/presets/client-preset)
+it is possible to write your GraphQL queries inlined, next to your component in a type-safe way.
+This can greatly improve the Developer Experience (DX).
```

Comment:
Maybe lets link to https://the-guild.dev/blog/unleash-the-power-of-fragments-with-graphql-codegen here?